packages/pn-commons/src/utils/__test__/notification.utility.test.ts

```diff
@@ -428,6 +428,17 @@ describe('notification status texts', () => {
       'status.canceled-description'
     );
   });
+
+  it('return notification status infos - CANCELLED - passing status only', () => {
```

Comment:
Replace CANCELLED with CANCELLATION_IN_ PROGRESS

Reply:
do it